### DISCUSSION TITLE
"Git commit message editor" looks ugly

### DISCUSSION THREAD

**Discussion by hzhangxyz** [syl20bnr/spacemacs#15405] [Q&A] [answered]
Hello, after update spacemacs(by git pull), my git commit message editor become ugly.

Here is what it looks like:
![image](https://user-images.githubusercontent.com/11623447/158058805-a64b7d94-0f22-4846-b23c-28a6f53f96ce.png)

every space is shown as "." and every "\n" is showln as "$", what should I do to recovery it?

My spacemacs version is (ed19c94a1).

And my .spacemacs is https://pastebin.com/mjMyf5rR

**Comment by lebensterben** [comment]
try to restart Emacs

**Reply by hzhangxyz** [reply]
> try to restart Emacs
I tried but it does not work.

**Comment by lebensterben** [comment, marked as answer]
It looks like you have show-whitespace-mode turned on.

**Reply by hzhangxyz** [reply]
Oh, thank you very much! There is really a "whitespace-mode" turned on, so how can I configure to disable it for this major mode "helm-ls-git-commit"? spacemacs/toggle-whitespace-globally-off/on is set as off, it seems this whitespace-mode is bound to the major mode.

**Reply by lebensterben** [reply]
I've no idea why it's on. We have not changed anything recently. It could be an upstream change but I need to investigate.

**Reply by hzhangxyz** [reply]
OK, Thank you. At least, I can disable it manually each time now.
